tools/osbuilder/rootfs-builder/nvidia/nvidia_rootfs.sh

```diff
@@ -165,7 +165,7 @@ setup_nvidia_gpu_rootfs_stage_one() {
 	mount -t proc /proc ./proc
 
 	chroot . /bin/bash -c "/nvidia_chroot.sh $(uname -r) ${run_file_name} \
-		${run_fm_file_name} ${machine_arch} ${NVIDIA_GPU_STACK}"
+		${run_fm_file_name} ${machine_arch} ${NVIDIA_GPU_STACK} ${KBUILD_SIGN_PIN}"
```

Comment:
Is `KBUILD_SIGN_PIN` always set here?

Reply:
No it is not, either it's "" empty or set with he password for enc/dec of the key.

Reply:
Is `set -eu` gonna kick off it's not set? Do we need `${KBUILD_SIGN_PIN:-}` here (or something else) was the question behind the question

Reply:
Indeed. this would've been catched wit shellcheck :) no puns intended.